content/hardware/02.hero/boards/uno-rev3/tutorials/PWM-using-MATLAB-Tutorial/PWM-using-MATLAB-Tutorial.md

```diff
@@ -0,0 +1,148 @@
+ 
+## Connecting the Board to MATLAB
+**1.** Connect your board to the computer via the USB 2.0 Cable Type A/B.
+
+![USB connection to computer](assets/Arduino-Connect1080.png)
+
+**2.** Open MATLAB and run the command `arduinosetup()` in the Command Window.
+
+![Arduino Setup](assets/arduinosetup.png)
+
+**3.** A Graphical User Interface (GUI) appears, that will help you set up the MATLAB®-Arduino connection. To establish the connection over the USB cable, make sure that the USB radio box is selected and then click on Next.
+
+![Selection of Connection Type](assets/connectiontype.png)
+
+
+**4.** Choose the "UNO" from the dropdown menu, as well as the relevant COM port, and click on the blue `Program` button to upload the Arduino Server to the board. When you have done so, click on next.
+
+![Programming the Arduino](assets/HardwareSetup1080.PNG)
+
+
+**5.** In this step you can review the connection type, Port, board and loaded libraries. You can also click on Test connection to evaluate the Arduino-MATLAB® connection. Next, you should see a Green check-mark signalling the successful connection, as shown in the image below.
+
+![Programming successful](assets/SetupSuccess1080.PNG)
+
+
+**6.** The UNO R3 is now configured to interact with MATLAB®. Now to create the object in MATLAB®, we run the command `a=arduino()`. The properties of the object, including the COM port, are displayed as shown in the example below.
+
+![Arduino properties](assets/arduinoproperties1080.png)
+
+**7.** Enter the command `writeDigitalPin(a,'D13',1);`. This command is similar to the `digitalWrite(D13, HIGH)` by the Arduino programming language. Which means that digital pin 13 (D13) is connected to the built-in LED, `a` is the Arduino object we created, and 1 represents a HIGH or ON state. Try turning the LED ON and OFF several times by changing 1 to 0 and vice-versa.
+
+
+***Unlike in the Arduino IDE, here you cannot address the built-in LED using the `LED_BUILTIN` function.***
+
+
+**8.** Now, you can use this command to continuously blink an LED as part of a `while` loop. Try entering the script below.
+
+```
+while (1)
+      writeDigitalPin(a, 'D13', 0);
+      pause(0.5);
+      writeDigitalPin(a, 'D13', 1);
+      pause(0.5);
+end
+```
+
+**9.** See what happens when you change the pause duration. Try replacing the while with an `if` statement to make it blink a certain number of times.
+
+## Circuit
+
+On one breadboard there is a white jumper wire sending 5 volts to the potentiometer. The orange cable is connected to GND(Ground) and the gray wire is our input to A0. A0 will read the variable voltage coming in from pin13 after being adjusted through the potentiometer.
+On the right breadboard we have a simple LED circuit comprised of a red output wire from pin11, a resistor and the LED finally connected to GND via the green wire.
+
+![Full circuit of the setup.](assets/circuit.png)
```

Comment:
@jhansson-ard We will need to replace this with an schematic, I'll send the draft